plugins/destination/mssql/client/columns.go

```diff
@@ -3,21 +3,19 @@ package client
 import (
 	"context"
 	"database/sql"
-	"strings"
 
 	"github.com/cloudquery/cloudquery/plugins/destination/mssql/queries"
 	"github.com/cloudquery/plugin-sdk/schema"
 	"github.com/cloudquery/plugin-sdk/specs"
-	"golang.org/x/exp/slices"
 )
 
 func (c *Client) pkEnabled() bool {
 	return c.spec.WriteMode == specs.WriteModeOverwrite || c.spec.WriteMode == specs.WriteModeOverwriteDeleteStale
 }
 
-func (c *Client) getTableColumns(ctx context.Context, table *schema.Table) (queries.Definitions, error) {
-	query, params := queries.GetTableSchema(c.schemaName, table)
-	var tc queries.Definitions
+func (c *Client) getTableColumns(ctx context.Context, table *schema.Table) (schema.ColumnList, error) {
```

Comment:
The changes here are related to me using `information_schema` to get the columns data instead of `sys.*`.
If you look at the changes to `plugins/destination/mssql/queries/table_info.go` you see the queries are shorter and easier to understand (in my opinion).
Also see https://www.mytecbits.com/microsoft/sql-server/information_schema-tables-vs-sys-tables